pkg/controller/workload/workload_stream_test.go

```diff
@@ -0,0 +1,301 @@
+/*
+ * Copyright 2024 The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package workload
+
+import (
+	"context"
+	"errors"
+	"net"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/agiledragon/gomonkey/v2"
+	config_cluster_v3 "github.com/envoyproxy/go-control-plane/envoy/config/cluster/v3"
+	discoveryv3 "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/test/bufconn"
+	"google.golang.org/protobuf/types/known/anypb"
+)
+
+type MockDiscovery struct {
```

Comment:
Can we import istio? I think we have done that for ads test

Reply:
This is because the workloadapi of kmesh is used in the package workload, and there is a conflict with istio.workload in the `TunnelProtocol`

```console
proto: file "api/workloadapi/workload.proto" has a name conflict over istio.workload.TunnelProtocol
```

Reply:
I've met this error during go build, and resolve it by https://github.com/kmesh-net/kmesh/pull/195/files

Not a must though